python/feldera/rest/feldera_client.py

```diff
@@ -188,18 +197,30 @@ def get_pipeline_stats(self, name: str) -> dict:
 
         return resp
 
-    def start_pipeline(self, pipeline_name: str):
+    def start_pipeline(self, pipeline_name: str, timeout_s: Optional[float] = 300):
         """
-        Start a pipeline
 
         :param pipeline_name: The name of the pipeline to start
+        :param timeout_s: The amount of time in seconds to wait for the pipeline to start. 300 seconds by default.
         """
 
+        if timeout_s is None:
+            timeout_s = 300

```

Comment:
Either the user is able to make it `None`, or it should just be a `float` -- whichever you prefer.